src/structures/Invite.js

```diff
@@ -71,22 +72,31 @@ class Invite extends Base {
     this.inviter = data.inviter ? this.client.users.add(data.inviter) : null;
 
     /**
-     * The target user for this invite
+     * The target user of this invite
      * @type {?User}
      */
     this.targetUser = data.target_user ? this.client.users.add(data.target_user) : null;
 
     /**
-     * The type of the target user:
+     * The target application of this invite
```

Comment:
```suggestion
     * The embedded application to open for this voice channel embedded application invite
```
from https://discord.com/developers/docs/resources/invite#invite-object-invite-structure